go.mod

```diff
@@ -1,10 +1,10 @@
 module k8s.io/website
 
-go 1.15
+go 1.16
 
 require (
 	k8s.io/apimachinery v0.20.0
-	k8s.io/kubernetes v1.20.0
+	k8s.io/kubernetes v1.21.0
```

Comment:
@reylejano , Are the go.mod changes related to the generated files in this PR?

Reply:
@kbhawkey , the generated files are with the go.mode changes with line 7 in go.mod `k8s.io/kubernetes v1.21.0`
But I did not change go.mod lines 11-37:

> replace (
	k8s.io/api => k8s.io/api v0.20.0
	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.20.0
	k8s.io/apimachinery => k8s.io/apimachinery v0.20.0
	k8s.io/apiserver => k8s.io/apiserver v0.20.0
	k8s.io/cli-runtime => k8s.io/cli-runtime v0.20.0
	k8s.io/client-go => k8s.io/client-go v0.20.0
	k8s.io/cloud-provider => k8s.io/cloud-provider v0.20.0
	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.20.0
	k8s.io/code-generator => k8s.io/code-generator v0.20.0
	k8s.io/component-base => k8s.io/component-base v0.20.0
	k8s.io/component-helpers => k8s.io/component-helpers v0.20.0
	k8s.io/controller-manager => k8s.io/controller-manager v0.20.0
	k8s.io/cri-api => k8s.io/cri-api v0.20.0
	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.20.0
	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.20.0
	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.20.0
	k8s.io/kube-proxy => k8s.io/kube-proxy v0.20.0
	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.20.0
	k8s.io/kubectl => k8s.io/kubectl v0.20.0
	k8s.io/kubelet => k8s.io/kubelet v0.20.0
	k8s.io/kubernetes => k8s.io/kubernetes v1.20.0
	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.20.0
	k8s.io/metrics => k8s.io/metrics v0.20.0
	k8s.io/mount-utils => k8s.io/mount-utils v0.20.0
	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.20.0
	k8s.io/sample-cli-plugin => k8s.io/sample-cli-plugin v0.20.0
	k8s.io/sample-controller => k8s.io/sample-controller v0.20.0
)

I can re-run with changes in lines 11-37 in go.mod from `v0.20.0` to `v0.21.0` and add to this PR

Reply:
The problem is that these modules are staged from the k/k repo. When `1.21.0` is not out, you won't get the `v0.21.0` tag from these modules.

Reply:
> The problem is that these modules are staged from the k/k repo. When `1.21.0` is not out, you won't get the `v0.21.0` tag from these modules.

@tengqm that's what I thought, but I re-ran with changes to go.mod to `v0.21.0 `or `v1.21.0`, I'll push but won't squash yet to see the changes